conf/distro/nilrt.conf

```diff
@@ -25,3 +25,23 @@ KERNEL_IMAGEDEST = "boot/runmode"
 ROOT_HOME = "/home/admin"
 
 IMAGE_LINGUAS ?= "en-us en-us.iso-8859-1"
+
+BUILD_IMAGES_FROM_FEEDS = "1"
+
+# Get NIOE packages from local deploy feed and NI packages from web feed
+IPK_NIOE_SUBFEED_URI ?= "file://${DEPLOY_DIR_IPK}"
+
+FEED_SRC ?= "download.ni.com/ni-linux-rt/feeds"
+NI_MAIN_FEED_VERSION ?= "2020.5"
+NI_SALT_FEED_VERSION ?= "20.0"
+IPK_NI_MAIN_SUBFEED_URI ?= "http://${FEED_SRC}/${NI_MAIN_FEED_VERSION}/ni-main/"
+IPK_NI_SALT_SUBFEED_URI ?= "http://${FEED_SRC}/SystemLink/salt/${NI_SALT_FEED_VERSION}/${MACHINE}/${TUNE_PKGARCH}/"
+
+# Configure feed URIs for image builds
+IPK_FEED_URIS += "\
+    NIOE-all##${IPK_NIOE_SUBFEED_URI}/all \
+    NIOE-${MACHINE}##${IPK_NIOE_SUBFEED_URI}/${MACHINE} \
+    NIOE-${TUNE_PKGARCH}##${IPK_NIOE_SUBFEED_URI}/${TUNE_PKGARCH} \
+    NI-software##${IPK_NI_MAIN_SUBFEED_URI} \
+    NI-salt##${IPK_NI_SALT_SUBFEED_URI} \
+"
```

Comment:
These lines can probably be moved to `nilrt.inc` as well. They fit well with the rest of the [OPKG SOURCE FEEDS](https://github.com/ni/meta-nilrt/blob/nilrt/master/dunfell/conf/distro/nilrt.inc#L112) section.